### DISCUSSION TITLE
/ref/log.html

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#8] [Announcements]
# /ref/log.html

VanBlog 的官方网站

https://vanblog.mereith.com/ref/log.html

**Comment by furxt** [comment]
不知道是我配置问题还是怎么着，页面一直报错！
Failed to fetch!

**Reply by Mereithhh** [reply]
这种情况下，一般是连不上数据库。您检查一下是不是改了编排的数据库账号密码，上面的没有同步改。
https://vanblog.mereith.com/guide/faq.html#%E9%83%A8%E7%BD%B2%E5%90%8E-http-error

如果还是不行可以提一个 issue 或者加交流群我帮您看。

**Comment by furxt** [comment]
您好！
    收到您的回复我感到很意外也很开心，至于您说的mongodb的数据库密码改动没有同步我应该注意了的，因为这个不同步的话，我好多功能都是非正常的(比如数据管理)；另外问题我也提了issue，您有时间可以看看，谢谢！



***@***.***
 
From: WangLu
Date: 2022-09-14 17:26
To: Mereithhh/vanblog-comment
CC: furxt; Comment
Subject: Re: [Mereithhh/vanblog-comment] /ref/log.html (Discussion #8)
这种情况下，一般是连不上数据库。您检查一下是不是改了编排的数据库账号密码，上面的没有同步改。
https://vanblog.mereith.com/guide/faq.html#%E9%83%A8%E7%BD%B2%E5%90%8E-http-error
如果还是不行可以提一个 issue 或者加交流群我帮您看。

**Reply by Mereithhh** [reply]
emmm，那个 issue 我看了，一般都是浏览器缓存导致的，不然你还是交流群私聊我吧。我需要了解具体的情况才好判断
